### PR TITLE
Add unlimited and throttled attributes

### PR DESCRIPTION
**Pull request by JonasSteur**
https://vikingco.atlassian.net/browse/INT-4936

### CODE REVIEW

File: definitions/subscriptions/balance.yaml [hunk 1]
```diff
@@ -40,6 +40,8 @@ properties:
         - remaining
         - used
         - type
+        - subtype
+        - status
```
Comment (freeduerinckx) [inline definitions/subscriptions/balance.yaml]: I though we settled on calling it `state` to keep naming in UWA more or less the same

Comment (JonasSteur) [reply definitions/subscriptions/balance.yaml]: Ah yes. I changed it below but forgot to update it here. Will fix

**Comment by freeduerinckx** [comment]
@nielsvanoch, @raul-mb  and @Kjoep can you have a look if these additions will fit the front end needs. Actually those are additions so everything should remain working as is, especially with regards to the old apps.

File: definitions/subscriptions/balance.yaml [hunk 2]
```diff
@@ -80,3 +82,15 @@ properties:
             - data
             - voice
             - value
+        subtype:
```
Comment (nielsvanoch) [inline definitions/subscriptions/balance.yaml]: If we're only going to use this for unlimited or not, I'd name the field specific to it. Perhaps even just a boolean called `unlimited`.

File: definitions/subscriptions/balance.yaml [hunk 3]
```diff
@@ -80,3 +82,15 @@ properties:
...
+          type: string
+          enum:
+            - default
+            - unlimited
+        state:
```
Comment (nielsvanoch) [inline definitions/subscriptions/balance.yaml]: Same here. If this field indicated throttled status, why not a boolean named `throttled`?

**Comment by JonasSteur** [comment]
@freeduerinckx @nielsvanoch After some more discussion we went for the boolean option. Can you re-review ?

File: definitions/subscriptions/usage.yaml [hunk 4]
```diff
@@ -47,4 +48,8 @@ properties:
   is_incoming:
     description: indicates if the call, data session or sms was incoming or not.
     type: boolean
-
+  throttled:
+    description: >
+      Indicates if throttling is active
```
Comment (nielsvanoch) [inline definitions/subscriptions/usage.yaml]: The meaning is `this CDR was generated while throttling was active`, right? Something along those lines might be more clear.

Comment (JonasSteur) [reply definitions/subscriptions/usage.yaml]: As this is the usage call that would be more clear indeed. Will amend.

**Pull request merged**